Jellyfin.Plugin.KodiSyncQueue/ScheduledTasks/RetentionTask.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Jellyfin.Plugin.KodiSyncQueue.Data;
+using MediaBrowser.Model.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Plugin.KodiSyncQueue.ScheduledTasks
+{
+    public class FireRetentionTask : IScheduledTask
+    {
+        private readonly ILogger _logger;
+
+        public FireRetentionTask(ILogger logger)
+        {
+            _logger = logger;
+            _logger.LogInformation("Retention Task Scheduled!");
+        }
+
+        public string Key => "KodiSyncFireRetentionTask";
+
+        public IEnumerable<TaskTriggerInfo> GetDefaultTriggers()
+        {
+            return new[] {
+
+                new TaskTriggerInfo
+                {
+                    Type = TaskTriggerInfo.TriggerDaily,
+                    TimeOfDayTicks = TimeSpan.FromMinutes(1).Ticks
+                }
+            };
+        }
+
+        public async Task Execute(CancellationToken cancellationToken, IProgress<double> progress)
+        {
+            //Is retDays 0.. If So Exit...
+            if (!int.TryParse(Plugin.Instance.Configuration.RetDays, out var retDays) || retDays == 0) {
+                _logger.LogInformation("Retention Deletion Not Possible When Retention Days = 0!");
+                return;
+            }
+
+            //Check Database
+            await Task.Run(() =>
+            {
+                retDays = retDays * -1;
+                var dt = DateTime.UtcNow.AddDays(retDays);
```

Comment:
```suggestion
                var dt = DateTime.UtcNow.AddDays(-retDays);
```
no need to multiply by `-1`